CHANGELOG.md

```diff
@@ -5,17 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
-
-### Added
-
-- Add reset button to disable integration heartbeat @mderynck ([#3959](https://github.com/grafana/oncall/pull/3959))
-- Schedule deleted integration cleanup task to run every 12 hours @mderynck ([#3976](https://github.com/grafana/oncall/pull/3976))
-
-### Fixed
-
-- Fix selecting values in notification settings ([#9566](https://github.com/grafana/support-escalations/issues/9566))
-- Allow setting integration_filter to null in webhooks internal API ([#4011](https://github.com/grafana/oncall/pull/4011))
```

Comment:
added the necessary `release:` labels to these PRs so that they will show up in the next release's release notes